contrib/nbfc-linux/files/nbfc

```diff
@@ -0,0 +1,8 @@
+# NoteBook FanControl service
+
+type               = process
+command            = /usr/bin/nbfc_service
+before             = login.target
```

Comment:
any particular reason to force it to start before login?

Reply:
nope, can be removed